help/en/releasenotes/current-draft-note.shtml

```diff
@@ -478,7 +478,7 @@
 
         <h4>Signal Masts</h4>
             <ul>
-                <li></li>
+                <li><li>
```

Comment:
No need to fix, but the second tag should be an end tag.

Reply:
Thanks!  That'll get cleaned out in the release process eventually.